test/unit/specs/field-types/MultiSelectFieldComponent.spec.js

```diff
@@ -40,6 +40,14 @@ describe('MultiSelectFieldComponent unit tests', () => {
                 value: 'ref1'
               }
             ])
+          } else if (Array.isArray(search) && search[0] === 'ref10') {
```

Comment:
maybe const the condition of the if with a name that describes the case being tested here